integrations/cloudbuild/build-all.yaml

```diff
@@ -5,8 +5,8 @@ steps:
       entrypoint: "./scripts/run_in_build_env.sh"
       args:
           [
-              "./scripts/build/build_examples.py --platform all build
-              --create-archives /workspace/artifacts/",
+              "./scripts/build/build_examples.py --platform all --create-flashbundle
```

Comment:
Any opinions on using `--enable-flashbundle` to somehow show it as 'optional if supported' ?